swaybar/status_line.c

```diff
@@ -85,6 +85,13 @@ bool status_handle_readable(struct status_line *status) {
 				}
 			}
 
+			json_object *float_coords;
+			if (json_object_object_get_ex(header, "float_coords", &float_coords)
```

Comment:
Maybe name this `float_event_coords`, to make it clear it's about events.

Reply:
Done.